app/core/models.py

```diff
@@ -0,0 +1,23 @@
+from django.db import models
+
+
+class Recipe(models.Model):
+    """Recipe object."""
+    name = models.CharField(max_length=255)
+    description = models.TextField(blank=True)
+
+    def __str__(self):
+        return self.name
+
+
+class Ingredient(models.Model):
+    """Ingredient for recipes."""
+    name = models.CharField(max_length=255)
+    recipe = models.ForeignKey(
```

Comment:
I'm not sure about it. One ingredient can only belong to only one Recipe? :thinking: 
I would do something like `ingredients = models.ManyToManyField('Ingredient')` in the Recipe model. What do you think?

Reply:
Yes I also agree. 😅  But this was a specific requirement mentioned in the exercise. That's why this model. Here is the requirement

<img width="781" alt="Screenshot 2022-09-27 at 09 22 15" src="https://user-images.githubusercontent.com/16999995/192460630-78c4f3ee-655b-4d12-98f5-e8bb9cc94ad0.png">

Reply:
Oh! I see. Ignore my comment then :+1: